Bottom: 5751c9dc71e09704b17ceea46b21cbbdb23ed0c6
Top:    4eebbbaed3ad33d744dc409ea175fbc628fe80cb
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-21 11:51:13 -0500

Update roles and privileges

Developers access to upload artifacts
Anonymous access to read artifacts


---
diff --git a/ansible-playbook/group_vars/all/nexus3.yml b/ansible-playbook/group_vars/all/nexus3.yml
index c11a953dd..be1c9bab9 100644
--- a/ansible-playbook/group_vars/all/nexus3.yml
+++ b/ansible-playbook/group_vars/all/nexus3.yml
@@ -43,6 +43,8 @@ ldap_connections:
     ldap_user_subtree: false
     ldap_group_subtree: true
 
+nexus_anonymous_access: true
+
 nexus_privileges:
   - name: all-repos-read  # used as key to update a privilege
     description: 'Read & Browse access to all repos'
@@ -56,8 +58,9 @@ nexus_roles:
     name: developers
     description: All developers
     privileges:
-      - nx-search-read
       - all-repos-read
+      - nx-repository-view-*-*-*
+      - nx-search-read
     roles: []
 
 nexus_local_users:
